low-level/files/opt/seagate/sspl/setup/setup_logger.py

```diff
@@ -27,11 +29,17 @@ def init_logging(syslog_host="localhost", syslog_port=514, console_output=False)
     """Initialize logging for sspl-setup."""
     _logger.setLevel(logging.INFO)
     # set Logging Handlers
-    handler = logging.handlers.SysLogHandler(
-                address=(syslog_host, syslog_port))
-    syslog_format = "%(name)s[%(process)d]: " \
-                    "%(levelname)s %(message)s (%(filename)s:%(lineno)d)"
-    formatter = logging.Formatter(syslog_format)
+    Utility().create_file(SETUP_LOG_PATH)
```

Comment:
Is file creation is really required ? will file not get created when you write log?

Reply:
@sandeepsgt  on the fresh VM, /var/log/cortx  dir won't exist. So need to configure the dir for the first time.
create_file() will check for complete path and create file if not exist.

Reply:
@gargpalak048 Can you please confirm that created directory/file have sspl-ll user permission?

Reply:
#ls -lrt  /var/log/cortx/sspl/sspl-setup.log
**-rw-r--r--** 1 root root 1031 Jul 12 02:16 /var/log/cortx/sspl/sspl-setup.log

file has 644 permissions @satish-seagate

Reply:
Whichever user runs miniprovisioner interfaces, file will be created with that user's permission.
Correct me if I am wrong, but I don't think this file need to have sspl-ll:sspl-ll user permission.

Reply:
@sandeepsgt may I know why permission is not required?

Reply:
`Whichever user runs miniprovisioner interfaces, file will be created with that user's permission.`
SSPL service has nothing to do with this file. So. I think permission is not required.

Reply:
Okay. @gargpalak048 @sandeepsgt I am okay with the change then.

Reply:
Better check if logs seen created properly in system where full Cortx Stck is deployed.

First dir /var/log/cortx creation should/would be done by Provisioner during deployment itself, though we should call that out as pre-requisite to provisioner if they are not doing it before component setup scripts are invoked.

Secondly dir needs to have write permission for other users too, beyond user creating that dir. Otherwise how sspl-ll user would be able to create sspl-setup log file then, check dir permission too here.

Reply:
If /var/log/cortx/sspl/ directory is not present, then it will be created with `drwxr-xr-x 2 root root` permission.
sspl_log file may not have permission issue with this, because sspl_log is created by rsyslog, not SSPL.

Reply:
I believe write permission for dir & file would already exists, including for sspl-ll user, otherwise it wouldnt be working today also. Though test it & confirm for sure